src/Microsoft.TestPlatform.PlatformAbstractions/net451/Tracing/PlatformEqtTrace.cs

```diff
@@ -29,25 +29,32 @@ public partial class PlatformEqtTrace : IPlatformEqtTrace
         public void SetupRemoteEqtTraceListeners(AppDomain childDomain)
         {
             Debug.Assert(childDomain != null, "domain");
-            if (childDomain != null && !Enum.Equals(TraceLevel, TraceLevel.Off))
+            if (childDomain != null)
             {
                 RemoteEqtTrace remoteEqtTrace = (RemoteEqtTrace)childDomain.CreateInstanceFromAndUnwrap(
                     typeof(RemoteEqtTrace).Assembly.Location,
                     typeof(RemoteEqtTrace).FullName);
```

Comment:
We need to create the `RemoteEqtTrace` object because we want to set the value of DoNotInitialize variable. 
If EqtTrace is not initialized in parent app domain, then we set `DoNotInitialize` in child app domain, will further stop instantiation of `TraceSource` which is a costly operation as it involves reading app.config file.